### PR TITLE
Added task realization

### PR DESCRIPTION
**Pull request by pkobevko**

### CODE REVIEW

File: src/main/java/practice/CandidateValidator.java [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+
+    private boolean checkPeriodsInUkr(Candidate candidate) {
+        int[] periodsInUkr = Arrays.stream(candidate.getPeriodsInUkr().split(HYPHEN_REGEX))
+                .mapToInt(Integer::parseInt)
+                .toArray();
+        return periodsInUkr[INDEX_OF_LAST_RESIDENCE_YEAR]
+                - periodsInUkr[INDEX_OF_FIRST_RESIDENCE_YEAR]
+                >= MIN_DURATION_OF_RESIDENCE_IN_UKRAINE;
+    }
     //write your code here
```
Comment (nechupara) [inline src/main/java/practice/CandidateValidator.java:30]: Remove this comment

**Review by nechupara — CHANGES_REQUESTED** [review]
good job! but let's improve your solution a little bit )

**Review by nechupara — APPROVED** [review]
Nice )

**Review by sarakhmen — APPROVED** [review]
Good job :)

**Pull request closed without merging**